components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+$$
+\frac{D {\bf u}_{3D} }{D t} \equiv
+\frac{\partial {\bf u}_{3D}}{\partial t}
++ {\bf u}_{3D}\cdot \nabla_{3D} {\bf u}_{3D}
+  = - \frac{1}{\rho} \nabla_{3D} p
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-rho)
+
+Geophysical fluids such as the ocean and atmosphere are rotating and stratified, and horizontal velocities are orders of magnitude larger than vertical velocities. It is therefore convenient to separate the horizontal and vertical as ${\bf u}_{3D} = \left( {\bf u}, w \right)$ and $\nabla_{3D} = \left( \nabla_z, d/dz \right)$ where $z$ is the vertical direction in a local coordinate frame on the Earth's surface and $w$ is the vertical velocity. The $z$ subscript on $\nabla_z$ is to remind us that this is the true horizontal gradient (perpendicular to $z$), as opposed to gradients within tilted layers used in the following section. The gravitational force $g {\bf k}$ is included in ${\bf F}^u$. The rotating frame of reference results in the Coriolis force $f {\bf k} \times {\bf u} \equiv f {\bf u}^\perp$, where $f$ is the Coriolis parameter and ${\bf u}^\perp$ is the horizontal velocity rotated $90^o$ from $\bf u$. See any textbook in the [References](#references) for a full derivation.
+
+The advection term may be separated into horizontal and vertical parts as
+
+$$
+{\bf u}_{3D}\cdot \nabla_{3D} {\bf u}_{3D}
+=
+{\bf u} \cdot \nabla_z {\bf u} + w \frac{\partial {\bf u}}{\partial z}.
+$$ (advection-3d2d)
+
+The horizontal component may be replaced with the vector identity
+
+$$
+\begin{aligned}
+{\bf u} \cdot \nabla_z {\bf u}
+&= (\nabla_z \times {\bf u}) \times {\bf u} + \nabla_z \frac{|{\bf u}|^2}{2} \\
+&= \left( \boldsymbol{k} \cdot (\nabla_z \times {\bf u})\right)
+\left( \boldsymbol{k} \times {\bf u} \right) + \nabla_z \frac{|{\bf u}|^2}{2} \\
+&= \zeta {\bf u}^{\perp} + \nabla_z K,
+\end{aligned}
+$$ (advection-identity)
+
+where $\zeta$ is relative vorticity and $K$ is kinetic energy. This step separates the horizontal advection into non-divergent and non-rotational components, which is useful in the final TRiSK formulation.
+
+#### Assumptions
+
+For a primitive equation ocean model, we assume the fluid is hydrostatic. For Omega we are not making the Boussinesq assumption, so all density-dependent terms use the full density. In particular, the density coefficient of the pressure gradient in [](#continuous-momentum) is not a constant, as it is in primitive equation models like POP and MPAS-Ocean.
+
+**Hydrostatic:** Beginning with the vertical momentum equation,
+
+$$
+\frac{D w }{D t}
+  =
+  - \frac{1}{\rho} \frac{\partial p}{\partial z} - g + {\bf D}^u + {\bf F}^u
```

Comment:
I think the forcing terms are now just the vertical components of those forcing vectors. I.e., ${\bf D}^{u} dot \bf k$ or $D^z$.

Reply:
Agreed. Done.